src/EPiServer.SocialAlloy.Web/Social/Models/ActivityStreams/PageRatingActivity.cs

```diff
@@ -1,24 +1,21 @@
-﻿using EPiServer.SocialAlloy.Web.Social.Adapters;
+﻿using EPiServer.Social.Common;
 
 namespace EPiServer.SocialAlloy.Web.Social.Models
 {
     /// <summary>
-    /// Represents the activity of a user rating being submitted in the site.
+    /// Represents the activity of a page rating being submitted in the site.
```

Comment:
maybe: the activity of a rating being submitted on a page in the site.
or the activity of a page being rated by a user navigating the site?